src/Microsoft.NetCore.Analyzers/Core/Performance/UsePropertyInsteadOfCountMethodWhenAvailable.cs

```diff
@@ -88,9 +88,8 @@ private void OnCompilationStart(CompilationStartAnalysisContext context)
         /// </summary>
         protected sealed class OperationActionsContext
```

Comment:
I think that the overall class might need a bit of cleaning (for simplification).